src/test/rustdoc/empty-section.rs

```diff
@@ -2,12 +2,12 @@
 
 #![feature(negative_impls)]
 
-pub struct Foo;
+pub struct Foo<T: ?Sized>(T);
```

Comment:
Why did this change?

Reply:
Because otherwise, the `Sized` trait impl will appear and make `// @!has - 'Auto Trait Implementations'` check fail.